notebooks/week6/01.send_request_to_endpoint.py

```
@@ -0,0 +1,135 @@
+# COMMAND ----------
+
+# Loop over skewed records and send requests for 30 minutes
+end_time = datetime.datetime.now() + datetime.timedelta(minutes=3)
+for index, record in enumerate(itertools.cycle(sampled_skewed_records)):
+    if datetime.datetime.now() >= end_time:
+        break
+    print(f"Sending request for skewed data, index {index}")
+    response = send_request_https(record)
+    print(f"Response status: {response.status_code}")
+    print(f"Response text: {response.text}")
+    time.sleep(0.2)
```

Ruff (0.7.0)

No newline at end of file

Add trailing newline

(W292)